app/logical/sites/definitions/commishes.yml

```diff
@@ -1,8 +1,10 @@
-type: SimpleDefinition
+type: ScraperDefinition
 enum_value: commishes
 display_name: Commishes
 homepage: https://commishes.com
 gallery_templates:
   - portfolio.commishes.com/user/{site_artist_identifier}
   - ych.commishes.com/user/{site_artist_identifier}
+  - ping.commishes.com/@{site_artist_identifier}
 username_identifier_regex: "[a-zA-Z0-9_-]{3,20}"
+submission_template: https://portfolio.commishes.com/upload/show/{site_submission_identifier}/
```

Comment:
Missing newline, then I'll merge

Reply:
Weird, VSCode is supposed to do that for me. It also inexplicably changed the line endings to CRLF (despite running on Arch Linux) but Rubocop at least caught that one before I pushed it. Either way, done.

Reply:
Should be caught by CI in the future https://github.com/Earlopain/reverser/pull/97